src/app/components/MediaLoader/configs/audio.ts

```diff
@@ -0,0 +1,50 @@
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import { ConfigBuilderProps, ConfigBuilderReturnProps } from '../types';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const { model: audioMediaBlock } = filterForBlockType(blocks, 'audio');
+  const { model: mediaOverrides } =
+    filterForBlockType(blocks, 'mediaOverrides') || {};
+  const audio = audioMediaBlock?.versions?.[0] || {};
+  const holdingImageURL = `https://${audioMediaBlock.imageUrl}`;
+
+  return {
+    playerConfig: {
+      ...basePlayerConfig,
+      autoplay: false,
+      statsObject: {
+        ...basePlayerConfig.statsObject,
+        episodePID: audioMediaBlock.id,
+      },
+      playlistObject: {
+        title: mediaOverrides?.pageTitleOverride,
+        holdingImageURL,
+        items: [
+          {
+            versionID: audio?.versionId,
+            kind: audioMediaBlock.smpKind,
+            duration: audio?.duration,
+          },
+        ],
+        summary: audioMediaBlock.synopses.short,
+        ...(audioMediaBlock.embedding && { embedRights: 'allowed' }),
```

Comment:
What's the rationale behind setting `embedRights: 'allowed'`
in `playlistObject` whereas the same is set to `'blocked'` in Morph?

See [config](https://github.com/bbc/morph-modules/blob/0f16bd34fad7339f0a61e5dcff3e489dbba9de3d/news-av-player/src/on-demand-audio/index.js#L70) for reference.

Reply:
This was probably because we had copied from TV, which in turn copied from the Clip media 😬 

TV, Radio & On Demand audio should never be embedded, so we should remove this. Could we assume that if we do not provide an external embed URL, and do not set the `embedRights` to a value, then the default might be `blocked` anyway, and therefore we have less config to worry about? 

As per my comment above, could we wait until we had the player on the page, and then add the settings which are required? Part of me wants to simplify this as much as possible.

Reply:
> This was probably because we had copied from TV, which in turn copied from the Clip media 😬
> 
> TV, Radio & On Demand audio should never be embedded, so we should remove this. Could we assume that if we do not provide an external embed URL, and do not set the `embedRights` to a value, then the default might be `blocked` anyway, and therefore we have less config to worry about?
> 
> As per my comment above, could we wait until we had the player on the page, and then add the settings which are required? Part of me wants to simplify this as much as possible.

So are we wanting to leave both of these until the MediaLoader component has been implemented?

Reply:
Yes. Let's do that.

Reply:
@Nabeel1276 let's remove the embedding config & update the tests accordingly. Then we can merge it in order to focus on putting the player on the page.

Reply:
So just `...(audioMediaBlock.embedding && { embedRights: 'allowed' }),` or only `&& { embedRights: 'allowed' }`?

Reply:
The entire line: 

```
...(audioMediaBlock.embedding && { embedRights: 'allowed' }),
```

Reply:
(which will never be true, so embedRights will never be allowed)